index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <nav class="navbar">
+      <div id="logo">
+        <a href="#">My Logo</a>
+      </div>
+      <div class="menu-items">
+        <ul>
+          <li><a href="#">Portfolio</a></li>
+          <li><a href="#">About</a></li>
+          <li><a href="#">Contact</a></li>
+        </ul>
+      </div>
+      <div id="burger">
+        <img src="/images/icon-Menu.svg" alt="burger_icon" />
+      </div>
+    </nav>
+ <section class="headline_section">
+<h1 class="primary_text">I’m Emery.
+Glad to see you!</h1>
+<p class="description">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+<h6 class="CTA">LET’S CONNECT</h6>
+ <img src="/images/social-media-icons.png" alt="social_icons" />
```

Comment:
- [ ] Excellent progress in adding the social media image tags! 
- [ ] To enhance the semantic structure, it would be beneficial to use a `ul` tag to group the list of icons, with each icon represented by a `li` tag. 
- [ ] Additionally, consider using an `a` tag for each icon, which will allow you to connect your social networks by utilizing the href attribute. 
- [ ] This approach not only ensures semantic correctness but also facilitates linking your icons with the respective social media profiles. Keep up the great work!
```suggestion
 <ul>
  <li><a href=""></a></li>
  <li><a href=""></a></li>
  <li><a href=""></a></li>
  <li><a href=""></a></li>
  <li><a href=""></a></li>
</ul>
```